about.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Oscafest</title>
+		<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <header>
+		<div class="header--social">
+			<ul>
+				<li><a href="#"><i class="bi bi-facebook"></i></a></li>
+        <li><a href="#"><i class="bi bi-twitter"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+			</ul>
+		</div>
+		<div class="header--nav--section">
+			<div class="header--hamburger">
+				<img src="images/hamburger.png" alt="hamburger-icon">
+			</div>
+			<a href="./index.html" class="header--logo">
+				<div class="logo--design">
+						<p>OS</p>
+				</div>
+				<p class="logo--text">OS Summit <br>2023</p>
+			</a>
+			<div class="header--nav--container">
+				<nav class="header--nav">
+					<img src="images/Icon.png" class="nav--close--icon" alt="close-icon">
+					<ul>
+						<li><a href="./index.html">Home</a></li>
+						<li><a href="#">Program</a></li>
+						<li><a href="#">Join</a></li>
+						<li><a href="#">Sponsor</a></li>
+						<li><a href="#">News</a></li>
+						<li><a href="#">Campaign</a></li>
+					</ul>
+				</nav>
+			</div>
+		</div>
+    <div class="about--hero--section">
+      <section class="about--header--section">
+        <p class="about--hero--greeting">"Hello! Sharing Wolrd"</p>
+        <h1>creative commons <br> global summit 2015</h1>
```

Comment:
- [ ] please kindly increase the size of the about page main heading and make it bolder as well, and use the correct color  `#EC5242` so it matches the design well.
- [ ] please work on your padding of contents, try as much as possible that they look almost identical to the original design as small ignored detail are not good when designing.
- [ ] kindly make the logo bigger, like in the original design. 
- [ ] l can see 3 different backgrounds, but we have just two in the original design. Kindly adjust that.
yours
![Screenshot (1189)_LI](https://github.com/Akorede3133/oscafest-conference/assets/65298048/55cc60c6-4737-4fcd-aefe-7ff62367fc63)
provided
![Screenshot (1188)_LI](https://github.com/Akorede3133/oscafest-conference/assets/65298048/a47f31bf-9b30-444a-8231-9d31613a1030)